config/routes.rb

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 
+BOT_ID = (ENV['BOT'] || :default).to_sym
```

Comment:
Maybe put that in `config/application.rb` or an initializer so that the default value isn’t repeated in different files?